corehq/apps/reports/templates/reports/filters/submit_error_types.html

```diff
@@ -4,7 +4,7 @@
 {% block filter_content %}
     <div id="{{ css_id }}" class="btn-group btn-group-separated report-filter-button-group" data-toggle="buttons-checkbox">
     {% for type in submission_types %}
-        <button class="btn btn-default{% if type.show %} btn-default{% else %} active{% endif %}" data-checkfilter="submission-filter-{{ forloop.counter }}">{{ type.name }}</button>
```

Comment:
There's similar logic in https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/filters/filter_users.html#L13 which also uses the `report-filter-button-group` class, does that template also need to be updated?

Reply:
How do I actually view that page on hq? I can trace the html back to this url: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/urls.py#L130, but it seems to be the url for the api?

Reply:
I think you're right that `UserTypeFilter` is never visible in main HQ, but it's used in custom reports: https://www.commcarehq.org/a/pact/reports/custom/chws/ And it does look like that report uses the same inverted behavior, where white means selected and purple means not selected.

You can't test custom reports locally. I'd just make the same changes to filter_user.html that you made to this file and leave yourself a note to test that report post-deploy.

Reply:
Sounds good, https://github.com/dimagi/commcare-hq/pull/23238/commits/43e5ac21742711c01673176214fdadd5d698215f
And I made a note to myself to check this after tomorrow's deploy